open_feature/open_feature_client.py

```diff
@@ -284,13 +281,18 @@ def evaluate_flag_details(
             )
 
             after_hooks(
-                flag_type, hook_context, flag_evaluation, reversed_merged_hooks, None
+                flag_type,
+                hook_context,
+                flag_evaluation,
+                reversed_merged_hooks,
+                hook_hints,
             )
```

Comment:
Seems like there are several common args among these. If you set this up with kwargs instead of args, you could do something like this, which (you make the call) could improve readability.

```python
common_hook_params = {'flag_type': ..., 'hook_context': ..., 'hook_hints': ...}
# ...
after_hooks(flag_evaluation, hook_context, **common_hook_params)
```

Reply:
These aren't immediately usable across all of the hooks, so while this might be able to reduce a few lines of code I don't think it's necessary to do.